pylint/pyreverse/writer.py

```diff
@@ -131,7 +131,11 @@ def get_values(self, obj):
         if not self.config.only_classnames:
             label = r"%s|%s\l|" % (label, r"\l".join(obj.attrs))
             for func in obj.methods:
-                label = r"%s%s()\l" % (label, func.name)
+                args = []
```

Comment:
Sweet, let's rewrite this with a comprehension:
```suggestion
                args = [arg.name for arg in func.args.args if arg.name != "self"]
```

Reply:
Awesome, will fix this later today.